distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -303,6 +304,23 @@ Text.match self pattern=".*" case_sensitivity=Case_Sensitivity.Sensitive =
     compiled_pattern = Regex.compile pattern case_insensitive=case_insensitive
     compiled_pattern.matches self
 
+## Compile the text into a regex pattern that can be used for matching.
+
+   Arguments
+   - case_insensitive: Enables or disables case-insensitive matching. Case
+     insensitive matching behaves as if it normalises the case of all input
+     text before matching on it.
```

Comment:
What does it do when set to `Nothing` (default)?

Reply:
It treats it as false; I vaguely remember there was a reason for this, but it doesn't make sense now. Changing it to False + removing the Nothing option.